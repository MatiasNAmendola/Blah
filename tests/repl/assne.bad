a:
12
b
:3
ad * 1

