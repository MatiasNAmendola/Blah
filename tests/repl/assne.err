^<parse>
^12$
^<eval>
^<parse>
^<eval>
