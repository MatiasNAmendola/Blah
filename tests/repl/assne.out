<parse> [Dk ASSN,Dn (Nd "a")] []
12
<eval> variable "b" does not exist
<parse> [Dk ASSN] [INT 3]
<eval> variable "ad" does not exist
